pages/ActivityStreams/Overview.md

```diff
@@ -32,23 +32,354 @@ These deviations from the standard will be thoroughly documented here.
+      "href": "https://github.com/example"
+    }
+  ]
+}
+```
+
+#### Video
+
+Clients should support video activity objects to represent video content posted by users.
+Video objects must include a `"type"` field with the value `"Video"`.
+Video objects must include a `"URL"` with either a URL string or an activity sub-object of type `"Link"` containing a URL pointing to a video file.
+Video objects must include a `"hash"` field with a [keccak-256](https://en.wikipedia.org/wiki/SHA-3) hash of the body of the linked video file.
+Video objects must include fields of `"height"` and `"width"` each with positive integer value on either the root activity object or the individual link sub-objects if multiple are present representing the height and width of the linked video file.
+Video objects must include a `"mediaType"` field on their link sub-object with a standard [MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml) string.
+
+Clients must support the following media types:
+
+| Format | MIME Type    | Specification(s)                                             |
+|--------|--------------|--------------------------------------------------------------|
+| MPEG   | `video/mpeg` | [RFC2045](https://www.iana.org/go/rfc2045)                   |
+| OGG    | `video/ogg`  | [RFC5334](https://www.iana.org/go/rfc5334)                   |
+| WebM   | `video/webm` | [WebM standard](https://www.webmproject.org/docs/container/) |
+
+Clients should support the following media types:
+
+| Format | MIME Type    | Specification(s)                           |
+|--------|--------------|--------------------------------------------|
+| H256   | `video/H256` | [RFC7798](https://www.iana.org/go/rfc7798) |
+| MP4    | `video/mp4`  | [RFC4337](https://www.iana.org/go/rfc4337) |
+| Raw    | `video/raw`  | [RFC4175](https://www.iana.org/go/rfc4175) |
+
+Video objects may include a `"duration"` field with a string complying with the [XML Schema 11-2](https://www.w3.org/TR/xmlschema11-2/) standard for duration strings as recommended in the [Activity Vocabulary](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-duration) specification.
+Video objects linking to live streaming content may use the string `"Live"` for the duration field or omit it entirely.
+
+For example, the following would be valid video objects:
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type": "Video",
+  "name": "Puppy Plays With Ball",
+  "url": "http://example.org/video.mkv",
+  "duration": "PT2H",
+  "width": 1024,
+  "height": 768
+}
+```
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type": "Video",
+  "name": "Big Buck Bunny",
+  "url": {
+    "type": "Link",
+    "href": "https://upload.wikimedia.org/wikipedia/commons/c/c0/Big_Buck_Bunny_4K.webm",
+    "width": 4000,
+    "height": 2250
+  },
+  "duration": "PT10M32S",
+  "mediaType": "video/webm"
+}
+```
+
+## Modifications and Extensions
```

Comment:
We don't discuss the absence of the "id" field. Should we?

Reply:
Added a section